### PR TITLE
Adds class 8

### PR DESCRIPTION
**Pull request by chrisbay**
This branch was created from `class7`, so it will probably be necessary to merge `master` after #9 has been merged.

### CODE REVIEW

**Comment by chrisbay** [comment]
[Migrate Liftoff class 8](https://trello.com/c/z2JYCetw/158-migrate-liftoff-class-8)

File: src/classes/8.rst [hunk 1]
```diff
@@ -14,12 +14,55 @@ Career Module
...
+Before we get started take a minute to have your standup meeting.
+
+Regroup: Intro to Continued Education Activity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For our final class today we want to talk about continued education. You
+must continue learning on your own after this class, and you must
+continue learning on the job. We will talk about how to do this.
+
+Activity: Continued Education
```
Comment (CarlyLanglois) [inline src/classes/8.rst:35]: this doesn't read to me as production ready for students to read. is there a decision yet to be made on what activity will be planned?

File: src/modules/project/deploying-java-apps.rst [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+created on the confirmation message.
+
+Preparing Your Project
+----------------------
+
+This article will be based off of the
+`cheese-mvc <https://github.com/LaunchCodeEducation/cheese-mvc>`__
+project used in the LC101 Java Track.
+
+.. note:: text
```
Comment (CarlyLanglois) [inline src/modules/project/deploying-java-apps.rst]: just flagging this incase it's a placeholder for something?

Comment (CarlyLanglois) [reply src/modules/project/deploying-java-apps.rst]: oops actually, i fixed it nvm

File: src/classes/8.rst (hunk 1 above)
Comment (CarlyLanglois) [reply src/classes/8.rst:35]: @chrisbay - just this to clarify then its ready to go

**Pull request merged**